index.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Psychological therapy Week Event 2023</title>
+  <link rel="icon" type="image/vnd.icon" href="./assets/img/favicon.ico">
+  <link rel="stylesheet" href="./assets/css/general.css">
+  <link rel="stylesheet" href="./assets/css/main.css">
+  <link rel="stylesheet" href="./assets/css/responsive.css">
+</head>
+
+<body>
+  <section id="mobile-menu" class="hidden box_flex flex_col justify_center gap_40">
+    <ul class="box_flex flex_col align_center gap_40">
+      <li><a href="about.html" class="mobile-nav-link" data-name="About">About</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="Program">Program</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="Join">Join</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="Sponsor">Sponsor</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="News">News</a></li>
+    </ul>
+    <div class="box_flex justify_center menu-footer-button">
+      <a href="#" class="btn_default primary_line_button">MH Campaigns</a>
+    </div>
+  </section>
```

Comment:
[Optional] I don't think that you need to have a separate navigation for the mobile version. The same mobile menu can be used for both and styled to hide elements that are not needed on a particular version.